CHANGELOG.md

```diff
@@ -2,6 +2,59 @@
 
 All notable changes to this project will be documented in this file.
 
+### [4.0.2](https://github.com/terraform-aws-modules/terraform-aws-lambda/compare/v4.0.1...v4.0.2) (2022-09-17)
```

Comment:
it looks like you have some changes to resolve - these should not be included in the changeset

Reply:
Thanks @bryantbiggs, I will have a look and resolve them.